domain/src/main/java/com/bottlerocketstudios/brarchitecture/domain/models/SnippetDetailsFile.kt

```diff
@@ -0,0 +1,7 @@
+package com.bottlerocketstudios.brarchitecture.domain.models
+
+data class SnippetDetailsFile(
+    val fileName: String?,
+    val links: Links?,
+    val rawFile: ByteArray? = null
```

Comment:
@MMurrayAtBR Can this be null per the backend api call response schema? If not, let's make this a non-null type